### ISSUE TITLE
Page Analyzer Run Complete: [M365Scc.ChartCard][Visual Requirements-Overview Page>Graphs]: Keyboard focus is not clearly visible on graph items

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/fluentui#17270]
User Experience:
Keyboard and low-vision users will not be able to detect the focused control if keyboard focus will not be visible on currently focused control

Test Environment:
OS build Version 2004 (OS Build 20262.1010)
Browser Version: Microsoft Edge Dev Version 89.0.723.0
URL: [https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/stackedbarchart](url)

Repro steps:
1. Open above URL on Anaheim Dev Browser and login with valid credentials.
2. Navigate to the 'Overview' tab by left arrow key and select it.
3. Navigate to 'Simulation Coverage' section by tab key and observe the issue

Actual Result:
Keyboard focus is not clearly visible on graph items, its just that the tooltip of the control gets pops-up when focused by keyboard

Expected Result:
Keyboard focus(black rectangular) boundary should be clearly visible on the the focused graph sections.

Observation: 
1.Issue does not reproe for more than one line graphs

NOTE:
Same issue reproes throughout the application for these one liner graphs

MAS Reference:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b3b98b04f-4fab-4d3e-81af-a0906a34849c%7d](url)

Attachments:
[MAS2.4.7-Keyboard focus is not clearly visible on graph items.zip](https://github.com/microsoft/fluentui/files/6081728/MAS2.4.7-Keyboard.focus.is.not.clearly.visible.on.graph.items.zip)

**Comment by gouttierre** [comment]
@Shubham786786 - Thank you for the feedback into the issue you are experiencing

@tomi-msft - Would you be able to confirm if this is a regression, or if this behavior is an issue?
There is another issue reported by the user with the same component in question (i.e. #17269). 
cc @smhigley
